frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/attached-devices.scss

```
@@ -71,7 +71,21 @@
 
 .ceph-ocs-install__filtered-modal {
   max-height: 75%;
-  width: 50%;
+  width: 55%;
+
+  // for this page, 100% width is required
+  .ceph-ocs-install__node-list {
+    max-width: 100%;
+  }
+
+  .co-m-nav-title {
+    margin-top: 0;
+  }
+
+  .co-m-pane__body {
+    padding-left: 0;
+    padding-right: 0;;
```

Nit: I think `co-m-nav-title` & `co-m-pane__body` should be inside the `ceph-ocs-install__filtered-modal` & we should not override the default class. 
```suggestion
    padding-right: 0;
```

---

I agree, but there is no other way to over write the styles. Here I have over-written the styles for just under `.ceph-ocs-install__node-list `, so it will not impact other views i.e. a > b

---

ack!